<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Que desea realizar?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="visualizar_imagenes">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ver imagenes de la carpeta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="datos_imagenes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ver datos sobre una 
 de las imagenes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="regresar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Regresar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
